### PR TITLE
0912mobileforms

### PR DESCRIPTION
**Pull request by sarahlshowers**

### CODE REVIEW

**Comment by derrickshowers** [comment]
Should we also adjust the form for mobile on the 'contact us' page?

File: app/wp-content/themes/kstg/library/scss/layout/_text-faq-page.scss [hunk 1]
```diff
@@ -93,3 +93,35 @@
...
+        }
+        .submit-wrap {
+          top: 170px;
+          left: -20px;
+        }
+      }
+    }
+
+    div {
+      #ninja_forms_field_3_div_wrap {
```
Comment (derrickshowers) [inline app/wp-content/themes/kstg/library/scss/layout/_text-faq-page.scss]: I guess this works--kind of limited on what you can do. I would change this to a class selector, though (e.g. `.textarea-wrap`). That way if he updates the form and the ID changes, it won't mess everything up.

**Comment by sarahlshowers** [comment]
asfjkhasdf

File: app/wp-content/themes/kstg/library/scss/layout/_text-faq-page.scss (hunk 1 above)
Comment (sarahlshowers) [inline app/wp-content/themes/kstg/library/scss/layout/_text-faq-page.scss]: Contact us is updated, it falls under the FAQ template... probably want to rename that to make it more generic?

**Pull request merged**